src/index.html

```diff
@@ -1,14 +1,56 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta
+      name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+    >
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="preconnect" href="https://fonts.googleapis.com" >
+    <link rel="preconnect" href="https://fonts.gstatic.com" >
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    >
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <div class="container">
+        <img src="./images/logo.png" alt="Moyo logo">
+
+        <nav>
+          <ul class="nav__list">
+            <li class="nav__item"><a class="nav__link is-active" href="#">Apple</a></li>
+            <li class="nav__item"><a class="nav__link" href="#">Samsung</a></li>
+            <li class="nav__item">
+              <a class="nav__link" href="#">Smartphones</a>
+            </li>
+            <li class="nav__item">
+              <a
+                class="nav__link"
+                href="#"
+                data-qa="hover">
+               Laptops & Computers
```

Comment:
the code style is still wrong. please pay more attention to the checklist (point 13)